ci-operator/step-registry/openshift/upgrade/qe/test/cert-rotation/openshift-upgrade-qe-test-cert-rotation-chain.yaml

```diff
@@ -9,6 +9,7 @@ chain:
   - ref: cucushift-upgrade-healthcheck
   - ref: cucushift-upgrade-toimage
   - ref: openshift-upgrade-qe-test-cert-rotation
+  - ref: openshift-oc-certrotation
```

Comment:
It doesn't make much sense to me to keep both of them. It's fine to use `openshift-oc-certrotation` only if it works fine in all the clusters. But I'm worried about if it will work with proxy cluster because I don't see the proxy setting in the step.

Reply:
To keep ref `openshift-upgrade-qe-test-cert-rotation` currently, just get one oc client with additional features for certrotation, I checked the latest oc of 4.11 nightly, not provided.

And I checked the rehearse-40684-periodic-ci-openshift-openshift-tests-private-release-4.11-amd64-nightly-4.11-upgrade-from-stable-4.11-vsphere-ipi-proxy-fips-cert-rotation-p2-f1 test, 
`Step vsphere-ipi-proxy-fips-cert-rotation-p2-f1-openshift-oc-certrotation succeeded after 14m54s. `
and rehearse-40684-periodic-ci-openshift-openshift-tests-private-release-4.10-amd64-nightly-4.10-upgrade-from-stable-4.10-vsphere-ipi-proxy-fips-cert-rotation-p2-f1 test,
`Step vsphere-ipi-proxy-fips-cert-rotation-p2-f1-openshift-oc-certrotation succeeded after 14m31s. `
We don't see what effect from proxy have on the test results of `oc-certrotation` step.

Reply:
We need to wait for Dev's PR https://github.com/openshift/release/pull/40536 to be merged before we evaluate if it will work with proxy cluster.

Reply:
`openshift-oc-certrotation` fails on 4.10 and lower versions because of `oc version` problem. The step `openshift-oc-certrotation` will fail on version 4.10 and below. Better create a separate chain.

Reply:
At present, the Dev's PR https://github.com/openshift/release/pull/40536 cannot be merged for the time being, I have to copy the PR code to our ref `openshift-upgrade-qe-test-cert-rotation` to execute the related upgrade profiles.